DependencyInjection/CompilerPass/FixturesCompilerPass.php

```diff
@@ -25,5 +25,7 @@ public function process(ContainerBuilder $container)
         }
 
         $definition->addMethodCall('addFixtures', [$fixtures]);
+
+        $definition->addMethodCall('addFixtureTagMapping', [$taggedServices]);
```

Comment:
I don't think it's feasible to add this mapping in a second call. IMO, this should be handled in the initial call to `addFixtures` or by creating multiple calls to `addFixture` and passing a list of sets/tags the fixture belongs to as second argument.